### PR TITLE
js added, skills section added, footer created

### PR DESCRIPTION
**Pull request by dartmater**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const today = new Date(); 
+
+const thisYear = today.getFullYear(); 
+
+const footerElement = document.querySelector("footer"); // footer element selected
+
+const copyright = document.createElement("p"); // copyright created
+
+
+copyright.innerHTML = `copyriight Elle Popova ${thisYear}`;
```
Comment (mannsn) [inline js/index.js:17]: misspelled copyright

File: css.css u [hunk 2]
```diff
@@ -0,0 +1,268 @@
+* {
```
Comment (mannsn) [inline css.css u:1]: Is this css used?

**Review by mannsn — APPROVED** [review]
Good job - check for extra files and misspelling of copyright - can fix in next submission (no need to resubmit this assignment).

File: css.css u (hunk 2 above)
Comment (dartmater) [reply css.css u:1]: no it was for the draft

**Pull request merged**